src/frontend/src/components/AddUser/index.tsx

```diff
@@ -127,14 +152,18 @@ class AddUser extends React.Component<Props, State> {
                 className="w-100 pa3 br2 b--black-20"
                 required={true}
                 aria-describedby={
-                  this.state.invalidCredentials
+                  this.state.missingPassword
+                    ? 'passwd_input_msg'
```

Comment:
I think using `password` instead of `passwd` in all instances of `passwd` would be a better practice. I am realizing that we are also using `msg` instead of `message` and am thinking we should do the same for that. These fields are being set for accessibility so it would be good to have them as readable as possible.

We will eventually want to change the `..._msg` entries in other components using a similar wording, but that will be saved for a different issue.

Reply:
Thanks for the feedback. I'll make the requested changes.